moveit_collisions_updater/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(moveit_collisions_updater)
+
+# Common cmake code applied to all moveit packages
+find_package(moveit_common REQUIRED)
+moveit_package()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(moveit_core REQUIRED)
+find_package(moveit_ros_planning REQUIRED)
+find_package(moveit_ros_visualization REQUIRED)
+find_package(rviz2 REQUIRED)
+find_package(rviz_common REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(urdf REQUIRED)
+find_package(srdfdom REQUIRED)
+find_package(ompl REQUIRED)
+
+
+# Tools Library
+add_library(${PROJECT_NAME}_tools
+  src/tools/compute_default_collisions.cpp
+  src/tools/moveit_config_data.cpp
+)
+target_include_directories(${PROJECT_NAME}_tools PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>
+  ${OMPL_INCLUDE_DIRS}
+)
+ament_target_dependencies(
+  ${PROJECT_NAME}_tools
+  "moveit_core"
+  "moveit_ros_planning"
+  "moveit_ros_visualization"
+  "rviz2"
+  "rviz_common"
```

Comment:
Why are these packages needed?

Reply:
Removed the rviz packages.  But had to leep the moveit_ros.  Ofterwise you get a ton of build errors.